GREEDY.UnitTests/OCRControllerUnitTest.cs

```diff
@@ -9,7 +9,9 @@ namespace GREEDY.UnitTests
     [TestClass]
     public class OCRControllerUnitTest
```

Comment:
If you uncomment it then we should change the name and project. Since this isin't a UnitTest it is wrong to keep it in UnitTests project and call the class UnitTest